interop/src/server.rs

```diff
@@ -114,27 +115,27 @@ impl pb::test_service_server::TestService for TestService {
                 let status = Status::new(Code::from_i32(echo_status.code), echo_status.message);
                 return Err(status);
             }
-
-            let single_message = stream::iter(vec![Ok(first_msg)]);
-            let mut stream = single_message.chain(stream);
-
-            let stream = try_stream! {
-                while let Some(msg) = stream.try_next().await? {
-                    if let Some(echo_status) = msg.response_status {
-                        let status = Status::new(Code::from_i32(echo_status.code), echo_status.message);
-                        Err(status)?;
-                    }
-
-                    for param in msg.response_parameters {
-                        tokio::time::delay_for(Duration::from_micros(param.interval_us as u64)).await;
-
-                        let payload = crate::server_payload(param.size as usize);
-                        yield StreamingOutputCallResponse { payload: Some(payload) };
-                    }
-                }
-            };
-
-            Ok(Response::new(Box::pin(stream) as Self::FullDuplexCallStream))
+            todo!();
```

Comment:
I think this is the same than [here](https://github.com/hyperium/tonic/pull/464/files#diff-b6467f7b50c468db706f5c04e0f3f73aR112)